### ISSUE TITLE
About Ano-AUC

### ISSUE THREAD

**Issue by kinder666** [nwpu-zxr/VadCLIP#9]
Hello Author, thanks for your great work. I have some questions about how to calculate the metric Ano-AUC，and is there any correlation between it and AUC1 and AUC2?

**Comment by nwpu-zxr** [comment]
ano-auc represents the auc result without normal samples in the testing set.

**Comment by MoshiraAbdalla** [comment]
Hello Author, where in the code the Ano-Auc ?
